workspace_analyzer.py

```diff
@@ -959,7 +989,7 @@ def isodd(x):
                     async_startup.append(
                         pool.apply_async(chunk_moller_trumbore,
                                          (verification_array,
-                                          stl_mesh.vectors[i * chunk_sz:, :],
+                                          tri_vecs[i * chunk_sz:, :],
                                           np.array([EPSILON, 0, 10 - EPSILON]),
```

Comment:
why 10?

Reply:
It's just a constant for the ray vector. Arbitrary, added as a constant with a comment explaining such.